perfkitbenchmarker/providers/gcp/gcp_dpb_dataflow.py

```diff
@@ -69,6 +70,15 @@ def SubmitJob(self, jarfile, classname, job_poll_interval=None,
                 job_arguments=None, job_stdout_file=None,
                 job_type=None):
     """See base class."""
+
+    if job_type == self.BEAM_JOB_TYPE:
+      full_cmd, beam_dir = beam_benchmark_helper.BuildMavenCommand(
+          self.spec, classname, job_arguments)
+      stdout, _, _ = vm_util.IssueCommand([full_cmd],
+                                          cwd=beam_dir,
+                                          use_shell=True, timeout=600)
```

Comment:
Is there a timeout in the integration test ? Does the IT fail if the timeout os breached ?